### PR TITLE
Add support for Klarstein Iceblock Ecosmart AC

### PR DESCRIPTION
**Pull request by krie**
Add support for Klarstein Iceblock Ecosmart AC

### CODE REVIEW

File: custom_components/tuya_local/devices/klarstein_iceblock_airconditioner.yaml [hunk 1]
```diff
@@ -0,0 +1,100 @@
+name: Klarstein Iceblock Ecosmart
+primary_entity:
+  entity: climate
+  dps:
+    - id: 1
```
[the remark this answers is not in the document]
Comment (krie) [reply custom_components/tuya_local/devices/klarstein_iceblock_airconditioner.yaml:8]: I would love to have to have the power switch combined with the modes, but unfortunately this causes issues with changing the mode via HA, when the device is already switched on. The device won't react to this command(s) then.

I guess it's the same problem as mentioned in https://github.com/make-all/tuya-local/discussions/1843. You answered there, that adding a separate switch would be the solution and that's why I did it here 😉

**Comment by krie** [comment]
Hi @make-all,
excuse my late reply, I was on vacation 🏖️
 
Thanks for your review and for merging my PR!
I will open a new PR to get into your comments.

File: custom_components/tuya_local/devices/klarstein_iceblock_airconditioner.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (krie) [reply custom_components/tuya_local/devices/klarstein_iceblock_airconditioner.yaml:8]: @make-all But the real question is, what would be the better solution for this? Having this kind of workaround separate switch or maybe change the way commands are send by your integration?

At least my device is apparently not the only one showing this behavior.

**Comment by krie** [comment]
Follow-up PR is https://github.com/make-all/tuya-local/pull/2096

**Comment by make-all** [comment]
Sorry, I jumped the gun making the change to separate out the power and modes again, causing a conflict, so I have made the changes manually.

**Pull request merged**